gen-nodejs/parquet_types.d.ts

```diff
@@ -1,5 +1,5 @@
 //
-// Autogenerated by Thrift Compiler (0.16.0)
+// Autogenerated by Thrift Compiler (0.18.1)
```

Comment:
I did regen these files with v0.18.1, but nothing was different except this header.